src/translations/en.translations.json

```diff
@@ -103,7 +103,7 @@
           "description": "All our tutorials, tool explanations, REX participation in community events and blog articles on various technologies in PHP web development."
         },
         "title": "Our articles and feedbacks about PHP programming",
-        "description": "<strong>PHP is a backend-oriented programming language</strong> that allows the development of dynamic and interactive applications. It is a popular language, used by a large number of web applications widely used by the general public. In this category, find all the articles, feedbacks and tutorials from our astronauts about <strong>PHP, Symfony, Laravel, the essential tools to improve your productivity, and our reports on major community events!</strong> Good reading!",
+        "description": "<strong>PHP is a backend-oriented programming language</strong> that allows the development of dynamic and interactive applications. It is a popular language, used by a large number of web applications widely used by the general public. In this category, find all the articles, feedbacks and tutorials from our astronauts about <strong>PHP, Symfony, Laravel, Angular, the essential tools to improve your productivity, and our reports on major community events!</strong> Good reading!",
```

Comment:
why is 'angular' mixed up with PHPs stuff here?

Reply:
La modification devait être fait sur Loco mais elle a été fait que FR apparament